packages/node_modules/brightspace-auth-token/README.md

```diff
@@ -72,7 +72,15 @@ The identifier for the acting user. For convenience, this will always be the
 same as `user` except in the case of impersonation. Not present outside of
 user context.
 
-___
+---
+
+#### `.azp` -> `String|Undefined`
+
+The identifier for the party to whom this token was issued. If present it **must** contain
+the OAuth 2.0 Client ID of the party. The identifier is a case-sensitive string which
+may be a URI value.
```

Comment:
Language suggestion, since this isn't really describing input to be accepted, but data that will is being presented.

```suggestion
The identifier for the party to whom this token was issued. If present it will contain
the OAuth 2.0 Client ID of the party. The identifier is a case-sensitive string which
may be a URI value.
```